### DISCUSSION TITLE
I want to change owner of organization

### DISCUSSION THREAD

**Discussion by break-jahun** [community/community#62005] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

The organization owner has left the company and can no longer log in to the github account.
That's why someone else needs to take over the owner's authority, but the owner can't be contacted.
I want a new person to get owner rights as soon as possible without the help of the original owner.
What can I do?

**Comment by nirav-gajera** [comment]
1. Make sure that you are an owner of the organization. If you are not an owner, you will not be able to transfer ownership.
2. Go to the organization's settings page.
3. In the "People" section, click "Add people."
4. In the "Add people" dialog box, enter the email address of the person who you want to give owner permissions to.
5. Select the "Owner" role from the drop-down menu.
6. Click "Add."
7. The person who you added will now have owner permissions to the organization.

If you are unable to contact the original owner, you can also try contacting GitHub support for assistance.

Here are some additional things to keep in mind when transferring ownership of a GitHub organization:

* The new owner will be responsible for all billing and administrative tasks for the organization.
* If the organization is using GitHub Enterprise Cloud, the new owner will need to be granted access to the organization's billing account.
* If the organization has any private repositories, the new owner will need to be granted access to those repositories.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @break-jahun , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
